test/app.test.js

```diff
@@ -0,0 +1,18 @@
+const request = require('supertest');
+const app = require('../lib/app');
+
+jest.mock('../lib/service.rickAndMortyApi.js');
```

Comment:
```suggestion
jest.mock('../lib/service/rickAndMortyApi.js');
```

Pathing error. After fixing this your app.test.js should pass.